website/docs/zombie-flags.mdx

```diff
@@ -0,0 +1,107 @@
+---
+
+Zombie flags are feature flags that are no longer needed or actively used. They often remain in the codebase long after their purpose has been fulfilled. Removing these outdated flags can reduce technical debt, improve performance, and simplify your codebase.
+
+Common scenarios where zombie flags appear:
+
+- A temporary feature becomes permanent, but its flag is never removed.
+- A flag was introduced for a one-time rollout or experiment that's now complete.
+- A decision was made to always keep a feature on or off, making the flag redundant.
+- The flag hasn't been toggled or referenced for an extended period.
+
+## Why Removing Zombie Flags Matters
+
+Leaving stale flags in your code can lead to the following:
+- **Increased complexity:** Unused flags clutter code and reduce maintainability.
+- **Unexpected behavior:** Outdated flags might trigger legacy logic unintentionally.
+- **Longer onboarding and reviews:** Developers waste time reviewing obsolete logic.
+- **Performance degradation:** In large-scale systems, excess flags can slow down operations.
+
+## Managing Zombie Flags in ConfigCat
+
+There are three ways to access and manage Zombie flags in ConfigCat:
+
+- Through the [Dashboard](https://app.configcat.com/product/zombie-flags).
+- Using the [Zombie Flags Email Report](https://app.configcat.com/my-account/zombie-flags-report).
+- Via the [Zombie Flags API](https://api.configcat.com/docs/index.html#tag/Zombie-(stale)-flags).
+
+### Using the Dashboard
+
+Using the Dashboard is the quickest way to view your zombie flags. Here's how to access the Zombie Flags page:
+
+1. In the [organization overview page](https://app.configcat.com/organization), select the product.
+2. Select **Zombie flags** from the product menu at the top of the page.
+
+<img alt="Navigating to Zombie Flags in the ConfigCat dashboard" src="/docs/assets/guide/my-zombie-flags-report/navigating-to-zombie-flags-in-the-dashboard_192dpi.png" width="1366" height="768"  decoding="async" loading="lazy" />
+
+The Zombie flags page shows all the stale flags in a table for the selected product based on the filters set:
+
+- **Inactivity**: How long the flag was inactive for.
+- **Feature flag scope**: All flags or only ones that are watched.
+- **Included environments**: The ConfigCat [environment](https://configcat.com/docs/main-concepts/#environment) for which you want to analyze flags. 
+- **Ignored flags based on tags**: The flags to exclude based on their tags.
+- **Stale in all/any selected environments**: Choose whether to include flags that are stale in all or any of the selected environments.
+- **Filter by config or feature flag name**: Type at least the first letter of the feature flag or config name to auto-filter the results.
+
+<img alt="Navigating to Zombie Flags in the ConfigCat dashboard" src="/docs/assets/guide/my-zombie-flags-report/the-zombie-flags-page-in-the-dashboard_192dpi.png" width="1366" height="768"  decoding="async" loading="lazy" />
```

Comment:
I think the alt tag is not correct here.

Reply:
I fixed the +tag button style, it looks awful in this small resolution. Could you please retake the screenshot after it is fixed? Thanks!